python311Packages.aiohttp-zlib-ng: 0.1.1 -> 0.1.2, python311Packages.cpufeature: init at 0.2.1

fabaff opened the pull request:
Diff: https://github.com/bdraco/aiohttp-zlib-ng/compare/v0.1.1...v0.1.2

Changelog: https://github.com/bdraco/aiohttp-zlib-ng/blob/v0.1.2/CHANGELOG.md

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
Result of `nixpkgs-review pr 275890` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.aiohttp-zlib-ng</li>
    <li>python310Packages.aiohttp-zlib-ng.dist</li>
    <li>python311Packages.aiohttp-zlib-ng</li>
    <li>python311Packages.aiohttp-zlib-ng.dist</li>
  </ul>
</details>

natsukium commented:
We need to add `cpufeature` to propagatedBuildInputs.
https://github.com/bdraco/aiohttp-zlib-ng/blob/9e381c8ec93051fa85df6f49c62c2106ce16daa9/pyproject.toml#L28

fabaff committed:
python311Packages.cpufeature: init at 0.2.1

Python module for detection of CPU features

https://github.com/robbmcleod/cpufeature

fabaff committed:
python311Packages.aiohttp-zlib-ng: 0.1.1 -> 0.1.2

Diff: bdraco/aiohttp-zlib-ng@v0.1.1...v0.1.2

Changelog: https://github.com/bdraco/aiohttp-zlib-ng/blob/v0.1.2/CHANGELOG.md

fabaff commented:
Result of `nixpkgs-review pr 275890` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.aiohttp-zlib-ng</li>
    <li>python310Packages.aiohttp-zlib-ng.dist</li>
    <li>python310Packages.cpufeature</li>
    <li>python310Packages.cpufeature.dist</li>
    <li>python311Packages.aiohttp-zlib-ng</li>
    <li>python311Packages.aiohttp-zlib-ng.dist</li>
    <li>python311Packages.cpufeature</li>
    <li>python311Packages.cpufeature.dist</li>
  </ul>
</details>

vcunat commented:
As shown on OfBorg already, this is broken on aarch64-linux.  Hydra: https://hydra.nixos.org/build/244719483

That broke home-assistant builds due to dependencies on `aiohttp-zlib-ng`.
